src/Discord.Net.WebSocket/Entities/Interaction/Message Components/SocketMessageComponent.cs

```diff
@@ -71,18 +68,21 @@ internal override void Update(Model model)
         }
 
         /// <inheritdoc/>
-        public override async Task RespondAsync(Embed[] embeds = null, string text = null, bool isTTS = false, InteractionResponseType type = InteractionResponseType.ChannelMessageWithSource,
```

Comment:
Why was the type removed? You should be able to specify the response type

Reply:
Because i make methods for all respose types

Reply:
Each of which is made so as to correspond to its own action